### PR TITLE
PSD-138: pipe flss loans into flss page

### PR DESCRIPTION
**Pull request by go-bears**
Jira ticket:
https://kiva.atlassian.net/browse/PSD-138

What does this do:
makes a graphql call to the flss endpoint in dev that's currently hardcoded to retail and renders loan cards to an /flss page
(localhost:888/flss)

How to test:
spin up local ui server
npm run dev -- --config=local and then can go to `localhost:8888/flss`

### CODE REVIEW

File: src/pages/FlssPrototypes/FlssLoanRow.vue [hunk 1]
```diff
@@ -223,19 +228,22 @@ export default {
...
 				fetchPolicy: 'network-only',
 			}).then(({ data }) => {
-				const newLoans = data.lend?.loans?.values ?? [];
+				const newLoans = data.fundraisingLoans?.values ?? [];
+
 				// Handle appending new loans to carousel
 				const newLoanIds = newLoans.length ? newLoans.map(loan => loan.id) : [];
-				const existingLoanIds = this.loans.length ? this.loans.map(loan => loan.id) : [];
+
+				const existingLoanIds = this.loans.length ? this.values.map(loan => loan.id) : [];
```
Comment (mcstover) [inline src/pages/FlssPrototypes/FlssLoanRow.vue:236]: I think this should be `this.loans.map(...` rather than `values`

**Comment by mcstover** [comment]
@go-bears Please checkout the latest version of `package-lock.json` from master to reset that in your branch.

File: src/graphql/query/flssQuery.graphql [hunk 2]
```diff
@@ -1,34 +1,21 @@
...
 					hash
 				}
-			}
-		}
-	}
+      }
+    }
 }
-
+s
```
Comment (bkimmig) [inline src/graphql/query/flssQuery.graphql:21]: This tabbing on this file looks a little off and there is an `s` at the end here - is that on purpose?

Comment (go-bears) [reply src/graphql/query/flssQuery.graphql:21]: hmm, maybe that happened during conflict/merge resolution in github's ui. b/c @mcstover 's git commit linting hooks wouldn't let me get by with that! Will lint and push again.

File: src/pages/FlssPrototypes/FlssLoanRow.vue [hunk 3]
```diff
@@ -75,19 +75,22 @@
...
-// import basicLoanQuery from '@/graphql/query/recommendedLoans.graphql';
+
 // import loanChannel from '@/graphl/query/loanChannelData.graphql'
+// import KvDropdown from '@/components/Kv/KvDropdown';
 import KvCarousel from '@/components/Kv/KvCarousel';
 import KvCarouselSlide from '@/components/Kv/KvCarouselSlide';
 import KvLoadingSpinner from '@/components/Kv/KvLoadingSpinner';
 import LoanCardController from '@/components/LoanCards/LoanCardController';
+// import flssLoanQuery from '@/graphql/query/flssQuery.graphql';
 import basicLoanQuery from '@/graphql/query/flssQuery.graphql';
```
Comment (mcstover) [inline src/pages/FlssPrototypes/FlssLoanRow.vue:86]: For the next stage of enhancement you'll want to rename these queries to match what their graphql file contains. `flssLoanQuery` in this case.

**Review by mcstover — APPROVED** [review]

**Review by mcstover — APPROVED** [review]

**Comment by go-bears** [comment]
thanks for the 👀  y'alls!

**Pull request merged**